src/containers/LocaleSelector.tsx

```diff
@@ -0,0 +1,16 @@
+import {ApplicationState} from '../types';
+import * as actions from '../actions/';
+import {connect} from 'react-redux';
+import LocaleSelector from "../components/menu/LocaleSelector";
+import {Dispatch} from "redux";
+
+
+export function mapStateToProps(state: ApplicationState) {
+    return {};
+}
+
+export function mapDispatchToProps(dispatch: Dispatch<actions.Action>) {
+    return {};
```

Comment:
I am by no means a React expert: Do we need this container, or can we just use the component directly?

Reply:
Yes, we do not need this container. Removed it!